include/Utils/DIPUtils.h

```diff
@@ -24,6 +24,21 @@
 
 #include "Utils/Utils.h"
 
+Value insertFMAOp(OpBuilder &builder, Location loc, VectorType type,
+                  Value input, Value kernel, Value output) {
```

Comment:
Please use `inputVec`, `'kernelVec`, `outputVec` here for clarity.

Reply:
Alright.